### DISCUSSION TITLE
File picker for markdown

### DISCUSSION THREAD

**Discussion by piotr-cz** [directus/directus#17803] [Feature Requests]
### Summary

Add file picker in markdown interface, similar to Image picker

### Motivation

Users should be able to add links to files in markdown text

### Detailed Design

Similar to Add image button

### Must Have:
- Option to add link to file

### Drawbacks

Not known

### Alternatives

Not known

### Adoption Strategy

Not known

**Comment by piotr-cz** [comment]
Pull request: https://github.com/directus/directus/pull/17804

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
